ports/geckolib/properties.mako.rs

```diff
@@ -184,6 +184,79 @@ def set_gecko_property(ffi_name, expr):
     }
 </%def>
 
+<%def name="convert_rgba_to_nscolor(rgba)">
+    {
+        let rgba = ${rgba};
+        (((rgba.alpha * 255.0).round() as u32) << 24) |
+        (((rgba.blue  * 255.0).round() as u32) << 16) |
+        (((rgba.green * 255.0).round() as u32) << 8) |
+         ((rgba.red   * 255.0).round() as u32)
+    }
+</%def>
+
+<%def name="convert_nscolor_to_rgba(color)">
+    {
+        use cssparser::RGBA;
+        let color = ${color};
+        RGBA {
+            red:    ((color        & 0xff) as f32) / 255.0,
+            green: (((color >>  8) & 0xff) as f32) / 255.0,
+            blue:  (((color >> 16) & 0xff) as f32) / 255.0,
+            alpha: (((color >> 24) & 0xff) as f32) / 255.0,
+        }
+    }
+</%def>
```

Comment:
I don't understand what we gain by codegen-ing these. Can we just make them regular rust functions in values.rs?